hack/nginx.conf

```diff
@@ -43,12 +43,13 @@ http {
 # Add as many IPs as desired.
 stream {
     upstream kubeendpoints {
-        server <server1 IP>:6443 max_fails=3 fail_timeout=10s;
-        server <server2 IP>:6443 max_fails=3 fail_timeout=10s;
+
+        # server <server1 IP>:6443 max_fails=3 fail_timeout=10s;
```

Comment:
These comments were always intended to be human read. That's why <server1 IP> is listed the way it is.

The load balancer won't work without adding upstreams here.

Reply:
They look pretty codey to be human read. Maybe worth adding some additional description surrounding usage, since it's not very clear to newbies like myself

Reply:
Added additional comment to help clarify.